bin/yaml/swift.yaml

```diff
@@ -31,6 +31,17 @@ compilers:
           - '5.4'
           - '5.5'
           - '5.6'
+    5.7-nightly:
+      if: nightly
+      type: restQueryTarballs
+      install_always: true
+      url: https://swift.org/builds/swift-5.7-branch/ubuntu2004/latest-build.yml
+      query: |
+        'https://swift.org/builds/swift-5.7-branch/ubuntu2004/' \
```

Comment:
you might be able to put `{name}` in place of the `{5.7}` here to make it a little more honest...I'll merge and check if that works ok! Thanks

Reply:
This def might be also useful for later, when a new stable branch appears and 5.7 is released officially, correct?

Reply:
Exactly! In this instance (I made the change and tested it) - we can just add a new `target: - 5.8` below and we're golden. Assuming nothing else changes :)

Thanks for the patch, I tested it locally. It'll install tonight -- I'm busy this weekend else I'd promise to get this and the other change deployed; maybe one of the other admins will get to it! Thanks for your contribution!

Reply:
Thank you!